packages/react-examples/src/react/PeoplePicker/PeoplePicker.Normal.Example.tsx

```diff
@@ -12,6 +12,7 @@ const suggestionProps: IBasePickerSuggestionsProps = {
   showRemoveButtons: true,
   suggestionsAvailableAlertText: 'People Picker Suggestions available',
   suggestionsContainerAriaLabel: 'Suggested contacts',
+  iconName: 'Delete',
```

Comment:
The changes in this file shouldn't be needed for the example to continue to work correctly, right? It'd be better not to change this example, to avoid confusing someone into thinking that they need to specify the icon name in the normal case.